DSpace/dspace-angular
# Fix embargoed files download link not working on edit item page

## References
* Fixes #1361

## Description
This fixes issue #1361 where you could not download an embargoed file from the edit item bitstream page.

## Instructions for Reviewers
* Created a new property in item-edit-bitstream.component.ts, bitstreamDownloadUrl: string
* The property gets populated on the ngOnChanges method, calling getBitstreamDownloadRoute on the page's current bitstream
* In the component's template the redirect is changed to this property, and the button is only shown if said property is not null

* To test the fix:
	- Login as an administrator
	- Submit a new item with an embargoed file
	- Navigate to edit item
	- Navigate to the "Bitstreams" tab
	- Click on the download icon for the embargoed file
	- The file should now download as expected

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
